main.tf

```diff
@@ -0,0 +1,117 @@
+# Declare some local variables
+locals {
+  build_environment = var.environment == "production" ? "Prod" : "Dev"
```

Comment:
glad we have a variable for this, but I don't see it being used anywhere and it's missing a "Sndbx" state - it should effect the name of each resource and be a common tag, the tag should have the full name "Sandbox", "Development", "Production".

Reply:
The environment variable configuration is complete. The github configuration for development is still pending due to the challenge we have with firewall / proxy.